users/vieting/experiments/switchboard/ctc/feat/baseline_args.py

```diff
@@ -142,6 +142,7 @@ def get_returnn_config(
     extra_args: Optional[Dict[str, Any]] = None,
     staged_opts: Optional[Dict[int, Any]] = None,
     audio_perturbation: bool = False,
+    useMultiProcDataset: bool = False,
```

Comment:
Please stay consistent with the style (no camel case).
```suggestion
    use_multi_proc_dataset: bool = False,
```